docs/docs/guide/_sections/binders/two-way.md

```diff
@@ -3,27 +3,28 @@ Two-way binders, like one-way binders, can update the DOM when a model property
 In order to update the model when the user interacts with the DOM, you need to tell Rivets.js how to bind and unbind to that DOM element to set the value on the model. Instead of defining the binder as a single function, two-way binders are defined as an object containing a few extra functions.
 
 ```javascript
-rivets.binders.toggle = {
-  bind: function(el) {
-    adapter = this.config.adapters[this.key.interface]
-    model = this.model
-    keypath = this.keypath
-
-    this.callback = function() {
-      value = adapter.read(model, keypath)
-      adapter.publish(model, keypath, !value)
-    }
+rivets.binders['mobi-calendar'] = {
```

Comment:
Isn't this quite a specific example? Should we keep the old `toggle` example too, as it is more generic?